File: codeforces/Sheet #1 (Data type - Conditions)/Java/H - Two numbers/H.java
import java.util.Scanner;
 
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        double A = sc.nextDouble();
        double B = sc.nextDouble();
 
        int floorResult = (int) Math.floor(A / B);
        int ceilResult = (int) Math.ceil(A / B);
        int roundResult = (int) Math.round(A / B);
 
        System.out.println("floor " + (int) A + " / " + (int) B + " = " + floorResult);
        System.out.println("ceil " + (int) A + " / " + (int) B + " = " + ceilResult);
        System.out.println("round " + (int) A + " / " + (int) B + " = " + roundResult);
    }
}

File: codeforces/Sheet #1 (Data type - Conditions)/python/H - Two numbers/H.py
import math
 
def custom_round(x):
    if x - math.floor(x) >= 0.5:
        return math.ceil(x)
    else:
        return math.floor(x)
 
A, B = map(float, input().split())
 
floor_result = math.floor(A / B)
ceil_result = math.ceil(A / B)
round_result = custom_round(A / B)
 
print(f"floor {int(A)} / {int(B)} = {floor_result}")
print(f"ceil {int(A)} / {int(B)} = {ceil_result}")
print(f"round {int(A)} / {int(B)} = {round_result}")
